### ISSUE TITLE
[Feature] Couple animator should control the other person's AO too

### ISSUE THREAD

**Issue by tonyasouther** [OpenCollar/opencollar#651]
Currently, the couples animation only attempts to turn the AO on and off for the wearer, using the sub AO interface. The Firestorm built-in AO will honor this protocol, so that much works.

This causes issues when the other person in the animation has an AO enabled, though, and it switches animations in the middle of the couple animation playing.

There is a standard LockMeister message to enable and disable an AO from any source. It's intended for messages from furniture and the like, but the Firestorm AO interface also supports it. Full documentation on the FS AO interface is at the [Firestorm wiki](http://wiki.phoenixviewer.com/fs_controlling_ao_w_scripts).

Basically, the player should disable the other person's animation when it disables the wearer's, and re-enable it when the wearer's is re-enabled.

**Comment by WendyStarfall** [comment]
I would love to see this, it would add a lot of convenience to the couple animator. Especially with modern AOs, it happens way too often that people hug one another and one of them suddenly wanders off a few meters as the AO switches to the next stand. Maybe we can make this also work with the AO Link.
